mtp_cashbook/templates/faq.html

```diff
@@ -0,0 +1,92 @@
+{% extends 'base.html' %}
+{% load i18n %}
+
+{% block page_title %}{{ view.title }} – {{ block.super }}{% endblock %}
+
+{% block body_classes %}{{ block.super }} mtp-credits{% endblock %}
+
+{% block content %}
+  {{ block.super }}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <a class="govuk-back-link" href="{% url 'home' %}">
+        {% trans 'Back' %}
```

Comment:
just remembered: for this link, it's actually better to do it in the view as `context['breadcrumbs_back'] = ???` so that it appears in the breadcrumb navigation bar rather than as part of the main page contents